tools/run_tests/run_xds_tests.py

```diff
@@ -1718,25 +1718,24 @@ def __init__(self, compute, alpha_compute, project):
                 metadata_to_send = '--metadata=""'
 
             if test_case in _TESTS_TO_FAIL_ON_RPC_FAILURE:
```

Comment:
This if block should be removed entirely as well

Reply:
Done